alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -0,0 +1,94 @@
+# PrometheusOperatorRejectedResources
+
+## Meaning
+
+The `PrometheusOperatorRejectedResources` alert triggers when Prometheus Operator
+rejects invalid `AlertmanagerConfig`, `PodMonitor`, `ServiceMonitor`, or `PrometheusRules`
+objects.
+
+## Impact
+
+The custom resources that trigger the alert are ignored by Prometheus Operator.
+As a consequence, they will not be part of the final configuration of the
+Prometheus, Alertmanager, or Thanos Ruler components managed by Prometheus Operator.
+
+## Diagnosis
+
+Find the custom resource type and the namespace in the `description` field
+of the `PrometheusOperatorRejectedResources` alert. You can find this information
+by using the OpenShift web console or the command line interface (CLI).
+
+### Diagnose using the Openshift web console
+
+1. Browse to **Observe** -> **Alerting**.
+2. If the **Filter** for **Alert State** is not set to **Firing**,
+ click the **Filter** drop-down list and select **Firing**.
+3. Search for the `PrometheusOperatorRejectedResources` alert.
+4. Click the alert to view its details.
+5. Scroll down and view the **Description** field. The namespace is displayed as
+a link, followed by the resource type.
+
+Knowing the namespace and the resource type, you can search for recently modified
+custom resources of that type in that namespace.
+However, note that the namespace label in the description can be either `openshift-monitoring`
+or `openshift-user-workload-monitoring`, but the faulty resource might still be
+located elsewhere.
+
+**Note**: If the namespace is named `openshift-monitoring`,
+this might be a bug in OpenShift. Please file a support case.
```

Comment:
I'd make the difference between openshift-monitoring and openshift-user-workload-monitoring more explicit.

```
The course of action depends on the value of the `namespace` label:
* When the value is `openshift-monitoring`, this is an issue with the platform monitoring stack. Please submit request to the support.
* When the value is `openshift-user-workload-monitoring`, this is an issue with a user-defined monitoring resource.
```